wolfssl/ssl.h

```diff
@@ -639,6 +639,15 @@ struct WOLFSSL_X509_STORE_CTX {
 
 typedef char* WOLFSSL_STRING;
 
+typedef struct WOLFSSL_RAND_METHOD {
+    int (*seed)(const void* seed, int len);
```

Comment:
Can you include some minor documentation for the use of each argument in these callbacks?

Reply:
Documentation added for all parameters.